### ISSUE TITLE
check-condition2 failure in Call.propagateExpectedType()

### ISSUE THREAD

**Issue by michaellilltokiwa** [tokiwa-software/fuzion#262]
this example, broken because `as_u64` does not take any args:
```
example =>
  (2).as_u64 1
```
yields:
```
java.lang.Error: check-condition2 failed!
        at dev.flang.util.ANY.check(ANY.java:390)
        at dev.flang.ast.Call.propagateExpectedType(Call.java:1643)
        at dev.flang.ast.Feature$5.action(Feature.java:1619)
        at dev.flang.ast.Feature$5.action(Feature.java:1617)
        at dev.flang.ast.Call.visit(Call.java:972)
        at dev.flang.ast.Call.visit(Call.java:49)
        at dev.flang.ast.Block.visit(Block.java:227)
        at dev.flang.ast.Block.visit(Block.java:41)
        at dev.flang.ast.Impl.visit(Impl.java:312)
        at dev.flang.ast.Feature.visit(Feature.java:1006)
        at dev.flang.ast.Feature.typeInference(Feature.java:1617)
        at dev.flang.ast.Resolution.resolveOne(Resolution.java:384)
        at dev.flang.ast.Resolution.resolve(Resolution.java:337)
        at dev.flang.fe.SourceModule.createASTandResolve(SourceModule.java:259)
        at dev.flang.fe.FrontEnd.<init>(FrontEnd.java:172)
        at dev.flang.tools.Fuzion.lambda$parseArgsForBackend$3(Fuzion.java:542)
        at dev.flang.tools.Tool.run(Tool.java:168)
        at dev.flang.tools.Fuzion.main(Fuzion.java:253)
```

**Comment by michaellilltokiwa** [comment]
#256 is a duplicate

**Comment by fridis** [comment]
fixed with f7eeaaf857e6e8b2b118c795aa9da1be0399c02c
